addons/account/wizard/account_invoice_send.py

```diff
@@ -33,26 +35,20 @@ class AccountInvoiceSend(models.TransientModel):
 
     @api.model
     def default_get(self, fields):
-        res = super(AccountInvoiceSend, self).default_get(fields)
+        res = super().default_get(fields)
         res_ids = self._context.get('active_ids')
 
         invoices = self.env['account.move'].browse(res_ids).filtered(lambda move: move.is_invoice(include_receipts=True))
         if not invoices:
             raise UserError(_("You can only send invoices."))
 
-        composer = self.env['mail.compose.message'].create({
-            'composition_mode': 'comment' if len(res_ids) == 1 else 'mass_mail',
-        })
-        res.update({
-            'invoice_ids': res_ids,
-            'composer_id': composer.id,
-        })
+        self.invoice_ids = res_ids
         return res
 
     @api.onchange('invoice_ids')
     def _compute_composition_mode(self):
         for wizard in self:
-            wizard.composer_id.composition_mode = 'comment' if len(wizard.invoice_ids) == 1 else 'mass_mail'
+            wizard.composition_mode = 'comment' if len(wizard.invoice_ids) == 1 else 'mass_mail'
```

Comment:
@JulienVR @william-andre A recent question from customer:

With this, the resulting messages in invoices in chatter is different either if sent on only 1 invoice and if sent with several invoices.

In the first case: the resulting message is considered as a mail (with the enveloppe icon)
In the second case: the resulting message is considered as internal note.

Could you explain that behaviour ?

Reply:
Hello @rousseldenis, this PR is outdated (it was not a good approach), another task is currently being developped by another developer.  Hopefully the Send & print wizard will be revamped soon :).

Reply:
Thanks, but that does not answer my question :smiley:

Reply:
I just tested and I have no idea why it behaves like that, I could not say whether it's a bug or deliberate, @william-andre doesn't know either. Maybe @rrahir ?

Reply:
As far a I recall, the idea was to mimic the behaviour of the action of sending an email either to one recipient or to a batch. That being said, it seems that this behaviour, if it ever existed, is no more. I guess we could simply drop it. Might be worth looking into it with one of the product owners. @chklop maybe ?

Reply:
I can not say if that's something intentional or not and it's not something that was changed since a long time.
As said by @JulienVR, we're investigating a complete revamp of the Send & Print in Odoo for future versions.